### PR TITLE
KDE report for 2022q3

### PR DESCRIPTION
**Pull request by adriaandegroot**

### CODE REVIEW

**Review by grahamperrin — COMMENTED** [review]
Some suggestions. 

Also, re: <https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#paragraphs> each sentence should begin on a new line. Thanks.

File: 2022q3/kde.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+==== Qt6 Landed
+
+The big news in the KDE ports is not directly KDE-related. Qt6 has landed, which prepares us for the next generation of Qt-based applications.
+It is now possible to have `USES=qt:6` to select the new Qt version. Some ports have been flavorized to make use of the new version.
+KDE itself is not affected: the port of KDE Frameworks to Qt6 is nearing completion, but is not yet completed. Most of them
+will compile with Qt6, but that is not important for FreeBSD ports. With the *devel/qt6* you get Qt 6.4.0, released at the end of the quarter.
+
+==== KDE Stack
+
+KDE Gear releases happen each quarter, KDE Plasma updates once a month, and KDE Frameworks have a new release each month as well.
```
Comment (pauamma) [inline 2022q3/kde.adoc:25]: s/each/every/g

File: 2022q3/kde.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+These (large) updates land shortly after their upstream release and are not listed separately.
+
+* KDE Frameworks 5 is now at version 5.98 (latest monthly release from September 2022).
+* KDE Gear is now version 22.08.1 (update for September 2022).
+* KDE Plasma is now version 5.24.6 (update for July 2022).
+
+Note that KDE Plasma 5.25 has been released upstream, but is waiting on fixes elsewhere in FreeBSD before it can land in the ports tree.
```
Comment (pauamma) [inline 2022q3/kde.adoc:32]: Can you elaborate on "elsewhere"? Do you mean ports it depends on, base FreeBSD, both?

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q3/kde.adoc (hunk 2 above)
Comment (grahamperrin) [reply 2022q3/kde.adoc:32]: Not a direct answer, but I'm aware of KDE bug [458356 – Plasma 5.25.x not launching on FreeBSD](https://bugs.kde.org/show_bug.cgi?id=458356).

Comment (pauamma) [reply 2022q3/kde.adoc:32]: Going to wait on the author answer, as this could be either as far as I can tell. Or it might be hard or impossible to tell.

Comment (adriaandegroot) [reply 2022q3/kde.adoc:32]: I was thinking "there's some PAM issue holding us back" but don't have a specific example there, Graham's upstream bug is a good one though that needs examination.

**Review by pauamma — APPROVED** [review]

**Pull request merged**